src/main/scala/it/pagopa/interop/catalogprocess/api/impl/Converter.scala

```diff
@@ -60,7 +60,7 @@ object Converter {
     attributes: Seq[AttributeManagementDependency.Attribute]
   ): Attributes = {
     val attributeNames: Map[String, AttributeDetails] =
```

Comment:
should it work if this becomes a `Map[UUID, AttributeDetails]` and we remove all added `toString`?